codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/OperationEndpointResolverMiddleware.kt

```diff
@@ -43,8 +43,7 @@ class OperationEndpointResolverMiddleware(
         ctx.service.getTrait<EndpointRuleSetTrait>()?.ruleSet?.let { node ->
             val ruleSet = EndpointRuleset.fromNode(node)
             val staticContextParams = op.getTrait<StaticContextParamsTrait>()?.parameters ?: emptyMap()
-            val clientContextParams =
-                ctx.service.getTrait<ClientContextParamsTrait>()?.parameters ?: emptyMap()
+            ctx.service.getTrait<ClientContextParamsTrait>()?.parameters ?: emptyMap()
```

Comment:
**Nit**: Looks like this does nothing and can be removed.

Reply:
fixed.